workspace/utilities/lib/popup.xsl

```diff
@@ -109,18 +110,18 @@
 				<xsl:with-param name="attr" select="$computed-attr-content"/>
 				<xsl:with-param name="content">
 
-					<xsl:call-template name="popup-close-button">
-						<xsl:with-param name="attr" select="$attr-close-btn"/>
-						<xsl:with-param name="url" select="$close-btn-url" />
-						<xsl:with-param name="content" select="$close-btn-content" />
-						<xsl:with-param name="is-optional" select="$close-btn-is-optional" />
-					</xsl:call-template>
-
-					<!-- content -->
+					<!-- Popup Content -->
 					<xsl:call-template name="content">
 						<xsl:with-param name="content" select="$content" />
 					</xsl:call-template>
-
+					<!-- Close button -->
+					<xsl:if test="/data/params/amp = 'No' and $display-close-button">
```

Comment:
Is the amp condition something we'd want to have in our template?

Reply:
Shouldn't we reverse the condition then to `/data/params/amp != 'no'` in order to display the button if amp is not in the project?

Reply:
@dominic-blain good catch, let me fix this.